github/PR-template

jsomsanith opened the pull request:
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [x] Other... Please describe: github PR template

**What is the current behavior?** (You can also link to an open issue here)
The PR template was not in root, so github did not  set the template by default


**What is the new behavior?**
Move to the root folder


**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...


**Other information**:

vjaouen commented:
IMHO, pull requests documentation must be technical only.

Hence, I would remove all functional / business description from the template because this information is already written in the JIRA and stick with technical validation only.

See my comments on the file

jsomsanith commented:
I added the jira link on top (and removed the comment to attach the jira in what's the current behavior).
I think that we should keep the `current behavor field` because there is never an explanation of the problem in the Jira (always the result) and sometimes it is necessary for a review to understand the problem from a technical point of view.
